src/main/java/handlers/CameraHandler.java

```diff
@@ -49,7 +49,9 @@ public CameraHandler(final InformationHandler information) {
      * @return The new camera as a Camera object.
      */
     public Camera addCamera(final String link) {
-        return addCamera(link, -1);
+        Camera cam = addCamera(link, -1);
+        cameraChestDetector.addCamera(cam);
+        return cam;
```

Comment:
If you would call the other addCamera method in stead of this one, is the camera then still added to the cameraChestDetector?